### ISSUE TITLE
Third parameter of partition has invalid type and results in false positives

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#470]
Fixed with #469.

Workaround, put the following in your `.clj-kondo/config.edn`:

```
{:linters
 {:type-mismatch
  {:namespaces
   {clojure.core
    {partition {:arities {2 {:args [:int :seqable]
                             :ret :seqable}
                          3 {:args [:int :int :seqable]
                             :ret :seqable}
                          4 {:args [:int :int :seqable :seqable]
                             :ret :seqable}}}}}}}}
```
